src/views/templates/details/tabs/scheduling/components/Descheduler.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { TemplateSchedulingGridProps } from 'src/views/templates/details/tabs/scheduling/components/TemplateSchedulingLeftGrid';
```

Comment:
using the full path can be a problem `src/...` , it is better to use relative paths instead like `../../......`
please change it for all your imports

Reply:
Well, I was told by Kobi that I should not use relative paths..

Reply:
what i said was the using `../` is not a nice solution, it indicate that the item you try to import is not the item you need.
here you have the same problem, using `src/...` indicate the same problem, you are trying to import something you should not need to import.
here you are trying to use a prop from a different component on the same level (`../` or `src/...` is has the same problem).

so if `Deschedualer` need props for `TemplateSchedualer...`  they should not be brothers (e.g. on the same level)
it means that  `TemplateSchedualer...` should be in some place other components that need it's prop can use without needing to use `../` or `src/...`.